### PR TITLE
Docker testing: handle licensing, different images per node

### PR DESCRIPTION
**Pull request by ncabatoff**

### CODE REVIEW

File: vault/external_tests/raft/raft_test.go [hunk 1]
```diff
@@ -464,10 +464,11 @@ func TestRaft_Configuration_Docker(t *testing.T) {
 				LogLevel: "TRACE",
 				// If you want the test to run faster locally, you could
 				// uncomment this performance_multiplier change.
-				//StorageOptions: map[string]string{
-				//	"performance_multiplier": "1",
-				//},
+				StorageOptions: map[string]string{
+					"performance_multiplier": "1",
```
Comment (cipherboy) [inline vault/external_tests/raft/raft_test.go:468]: Was this intentional?

**Review by cipherboy — COMMENTED** [review]
Just curious what the separate image per node does? Are you looking to test heterogeneous cluster deployments (e.g., mix of ubi/alpine)?

File: sdk/helper/testcluster/docker/environment.go [hunk 2]
```diff
@@ -501,6 +469,8 @@ type DockerClusterNode struct {
 	RealAPIAddr          string
 	ContainerNetworkName string
 	ContainerIPAddress   string
+	ImageRepo            string
```
Comment (cipherboy) [inline sdk/helper/testcluster/docker/environment.go:472]: Am I missing what sets `ImageRepo` now?

File: vault/external_tests/raft/raft_test.go (hunk 1 above)
Comment (ncabatoff) [reply vault/external_tests/raft/raft_test.go:468]: Nope, ty.

File: sdk/helper/testcluster/docker/environment.go (hunk 2 above)
Comment (ncabatoff) [reply sdk/helper/testcluster/docker/environment.go:472]: I think so, addNode should be the one.

**Review by cipherboy — APPROVED** [review]

**Pull request merged**